guides/akka-cluster-kubernetes-k8s-deploy/akka-cluster-example/build.sbt

```diff
@@ -0,0 +1,74 @@
+import sbt._
+import com.typesafe.sbt.SbtScalariform.ScalariformKeys
+import de.heikoseeberger.sbtheader.HeaderPattern
+import scalariform.formatter.preferences.AlignSingleLineCaseStatements
+import com.typesafe.sbt.packager.docker._
+
+lazy val akkaVersion = "2.5.4"
+
+name := "myapp"
+
+scalaVersion := "2.11.11"
+
+enablePlugins(AutomateHeaderPlugin, JavaServerAppPackaging)
+
+resolvers += Resolver.bintrayRepo("hseeberger", "maven")
+
+libraryDependencies ++= List(
+  "com.typesafe.akka" %% "akka-actor"          % akkaVersion,
+  "com.typesafe.akka" %% "akka-cluster"        % akkaVersion,
+  "com.typesafe.akka" %% "akka-http"           % "10.0.10",
+  "de.heikoseeberger" %% "akka-http-play-json" % "1.19.0-M2",
+  "com.typesafe.play" %% "play-json"           % "2.6.5",
+  "org.scalatest"     %% "scalatest"           % "3.0.1"     % "test",
+  "com.typesafe.akka" %% "akka-testkit"        % akkaVersion % "test"
+)
+
+dockerEntrypoint ++= Seq(
+  """-DakkaActorSystemName="$AKKA_ACTOR_SYSTEM_NAME"""",
+  """-Dakka.remote.netty.tcp.hostname="$(eval "echo $AKKA_REMOTING_BIND_HOST")"""",
+  """-Dakka.remote.netty.tcp.port="$AKKA_REMOTING_BIND_PORT"""",
+  """-Dakka.cluster.seed-nodes.0="akka.tcp://${AKKA_ACTOR_SYSTEM_NAME}@${AKKA_SEED_NODE_HOST_0}:${AKKA_SEED_NODE_PORT}"""",
+  """-Dakka.cluster.seed-nodes.1="akka.tcp://${AKKA_ACTOR_SYSTEM_NAME}@${AKKA_SEED_NODE_HOST_1}:${AKKA_SEED_NODE_PORT}"""",
+  """-Dakka.cluster.seed-nodes.2="akka.tcp://${AKKA_ACTOR_SYSTEM_NAME}@${AKKA_SEED_NODE_HOST_2}:${AKKA_SEED_NODE_PORT}"""",
```

Comment:
Thoughts on using Chirper style here instead -- that is, have the only environment variable be a list of seed nodes, thus supporting any number of seed nodes the user wishes? Up to you but I think it would be a bit nicer since it's more general.

This magic would help implement:

`$(IFS=','; I=0; for NODE in $AKKA_SEED_NODES; do echo "-Dakka.cluster.seed-nodes.$I=akka.tcp://$AKKA_ACTOR_SYSTEM_NAME@$NODE"; I=$(expr $I + 1); done)`

If proceeding, the guide would have to updated as well.

Reply:
That's a good idea - updated to match Chirper style now.